bitnami/rabbitmq/templates/statefulset.yaml

```diff
@@ -280,6 +287,12 @@ spec:
               - key: tls.key
                 path: server_key.pem
         {{- end }}
+        {{- if .Values.rabbitmq.communityPlugins }}
+        - name: {{ template "rabbitmq.fullname" . }}-plugins
+          configMap:
+            name: {{ template "rabbitmq.fullname" . }}-plugins
+            defaultMode: 0777
```

Comment:
Why is this needded? I think the configmap only needs execution and read permissions

Reply:
I agree; that's a leftover from things I tried